contents/docs/migrate/plausible.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Migrate from Plausible to PostHog
+sidebar: Docs
+showTitle: true
+---
+
+import MigrationWarning from "./_snippets/migration-warning.mdx"
+
+<MigrationWarning />
+
+Compared to other tools, Plausible is easy to get data out of, but it is difficult to convert to PostHog's schema. This is because data exported from Plausible is aggregated, rather than the individual events that make up the aggregation (like we expect in PostHog).
+
+This means we can try to convert data, but it won't be 100% accurate. For example, Plausible data tells there is a certain number of visitors, pageviews, and visits, but it doesn't tell you which user did what event at what time. We need break down this data to generate individual events that create an equivalent aggregation.
+
+This guide aims to convert aggregate data into events where possible and capture them in PostHog. 
+
+> **Terminology differences:** Although pageviews, custom events, and many properties are the same between Plausible and PostHog, there are two major terminology differences:
+> 1. Plausible uses **visitors** to represent the users behind pageviews and custom events. In PostHog, these are called [persons](/docs/data/persons).
+> 2. Plausible uses **variousisit** to represent a single use of your product or visit to your site. In PostHog, these are called [sessions](/docs/data/sessions).
+
+## 1. Export data from Plausible
+
+To export data from Plausible, go to your site settings and click on the **Imports & Exports** tab. Once here, click **Prepare download** and download the collection of `csv` files once ready.
+
+![Plausible exports](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_10_14_at_12_07_03_e73df931df.png)
+
+## 2. Convert Plausible data and capture it in PostHog
+
+> **🛠️ Work in progress:** The follow import converts and capture visitors and pageviews, but is not comprehensive. For example, it doesn't include:
+> 1. Person or event properties like UTMs or channels
+> 2. Session data (generating `$session_id` and correct timing for session duration)
+> 3. Bounce rate
+> 4. Custom events
+> 5. Entry or exit pages
+>
+> Contributions are welcome.
```

Comment:
We should say how they can contribute e.g. link to repo, open an issue, send an email, etc.